### PR TITLE
Spoofing [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by akashrajkn**

### CODE REVIEW

File: forward43/kickstarter_scraper.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        headers_    = {
+            'User-Agent': user_agent_,
+            'Accept': accept_,
+            'Accept-Language': 'en-US,en;q=0.5',
+            'DNT': '1',
+            'Connection': 'keep-alive',
+            'Upgrade-Insecure-Requests': '1',
+        }
+
+        r = requests.get(url, headers=headers_, proxies=proxies)
```
Comment (andrewsutjahjo) [inline forward43/kickstarter_scraper.py:58]: Nice one! 
completely non-need to have, but if you find yourself with nothing to do:  bonus points if you abstract this out as a wrapper for requests.get for other scrapers to use :)

File: forward43/kickstarter_scraper.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+ATTRIBUTES                 = ['rank', 'id', 'name', 'creator', 'goal', 'pledged',
+                              'state', 'backers', 'launch_date', 'deadline', 'url']
+
+def write_data_to_file(data, category):
+    '''
+    Write json data to file
+    @data: dictionary
+    @category: category id
+    '''
+    filepath = '../data/{}.json'.format(category)
```
Comment (andrewsutjahjo) [inline forward43/kickstarter_scraper.py:29]: Same comment as the other PR: 
```
from forward43.data.data_path import DATA_DIRECTORY
```
could help make this a bit sturdier :)

File: forward43/kickstarter_scraper.py (hunk 1 above)
Comment (akashrajkn) [reply forward43/kickstarter_scraper.py:58]: :+1:  I'm in process of making a generic scraper class with methods for data handling, processing, etc. I will include spoofing there as well.

Comment (andrewsutjahjo) [reply forward43/kickstarter_scraper.py:58]: 👍

**Comment by akashrajkn** [comment]
Moved this functionality to Scraper class in PR #12 (feedback addressed)

**Pull request closed without merging**